sympy/core/sympify.py

```diff
@@ -266,6 +266,17 @@ def sympify(a, locals=None, convert_xor=True, strict=False, rational=False,
     except AttributeError:
         pass
 
+    #Support for basic numpy datatypes.
+    if type(a).__module__ == 'numpy':
+        import numpy
+        if(numpy.isscalar(a)):
+            try:
+                return sympify(numpy.asscalar(a))
+            except RuntimeError:
+                return sympify(numpy.float64(a))
```

Comment:
But SymPy has `Float`, which is arbitrary precision. We should probably create a Float with matching precision as the numpy float.